doc/nrf/gs_recommended_versions.rst

```diff
@@ -0,0 +1,23 @@
+.. _gs_recommended_versions:
+
+Required Tools
+##############
+
+The following table show the list of tools required for working with |NCS| **v1.3.0**.
+It lists the minimum required version as well as the version that is installed when installing using the :ref:`gs_app_tcm` as described in :ref:`nRF Connect for Desktop <gs_assistant>`.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tool
+     - Minimun Version
+     - Toolchain Manager Version
+   * - Python
+     - |python_min_ver|
+     - |python_recommended_ver|
+   * - GNU ARM Embedded
+     - |gnuarmemb_min_ver|
+     - |gnuarmemb_recommended_ver|
+   * - SES - Nordic Edition
```

Comment:
```suggestion
   * - SEGGER Embedded Studio (Nordic Edition)
```

Reply:
Did that on first version, but it looked bad when rendering the page locally, due to the first column became too wide.

Reply:
But that is the name of the product - so we don't have a choice. And content is more important than how it looks. ;)

Reply:
This is how it looks:
![image](https://user-images.githubusercontent.com/9802583/85265538-b5f19000-b472-11ea-85f4-88ea13afcd35.png)
a horizontal ruler, and the `n` cut off per default.
Alternative, do you know a trick to wrap inside a cell, so we could have:
```
+-------------------------+----------+----------+
| SEGGER Embedded Studio  | 4.52     | 4.52     |
| Nordic Edition          |          |          |
+-------------------------+----------+----------+
```

Regarding `SEGGER Embedded Studio` vs `SES`, we are using SES in the docs already: 
Both in the TOC:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/getting_started.html?highlight=ses
and in the `Building and Programming`
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_programming.html?highlight=ses

Reply:
Yes, we are using SES as abbreviation after we have introduced it earlier on in the topic. The first occurrence must be spelled out.

And this is just how tables look in RST. Seems you're using a very small window though (or haven't updated to the latest stylesheet that uses full width).

Reply:
Nope.
Using the css from master, and a big window, notice the grey area left of the rendered page until window frame:
![image](https://user-images.githubusercontent.com/9802583/85270258-534fc280-b479-11ea-902f-7b30a1c2b9de.png)

Note, i'm using Firefox 77.0.1 on Linux.

Reply:
> The first occurrence must be spelled out.

Then the header should be changed here:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_programming.html#building-with-ses

cause people might see the TOC before reading the text inside this section.
And in the TOC then SES has not been spelled out.

Reply:
> Using the css from master, and a big window, notice the grey area left of the rendered page until window frame:

A complete clean of doc-build, and rebuilding from scratch fixed the grey area.

Reply:
Just trust the tech writer on when abbreviations are okay and when not. ;-) In subheadings it's fine, in the table we should spell it out.

How does the output on https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/index.html look for you? Same space?

Reply:
I do trust tech writers, just wanted to ensure that having SES in TOC was not something that had slipped by the reviewers eyes by mistake.